arkh/arkh.json

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "Arkh",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Arkhadian",
+    "chain_id": "arkh",
+    "bech32_prefix": "arkh",
+    "daemon_name": "arkhd",
+    "node_home": "$HOME/.arkhd",
+    "genesis": {
+        "https://github.com/vincadian/arkh-blockchain/blob/master/genesis/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "uarkh",
+                "fixed_min_gas_price": 0.000000001
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
+        
+    },
+    "peers": {
+        "seeds": [
+            {
+                "id": "808f01d4a7507bf7478027a08d95c575e1b5fa3c",
+                "address": "asc-dataseed.arkhadian.com:26656"
+            },
+            
+        ],
+        "persistent_peers": [
+           
+        ]
+    },
+    
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://asc-dataseed.arkhadian.com/",
+                "provider": "arkhnetwork"
+            },
+
+        ],
+        "rest": [
+            {
+                "address": "https://asc-blockchain-api.arkhadian.com/",
+                "provider": "arkhnetwork"
+            }
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "Ping",
+            "url": "https://testnet.ping.pub/arkh",
+            "tx_page": "https://testnet.ping.pub/arkh/blocks"
```

Comment:
This is the blocks url, not the txs url.  If there is no txs page, that's okay, just remove this line

Reply:
I think the ping.pub tx url is something like this:

`https://ping.pub/osmosis/tx/${txHash}`

Can you confirm that your explorer has one like this

Reply:
```suggestion
            "tx_page": "https://testnet.ping.pub/arkh/tx/${txHash}"
```